### DISCUSSION TITLE
FL Studio distortion problem with multiple channels bug (Found the reason, in other daw all fine)

### DISCUSSION THREAD

**Discussion by videohosting** [apohl79/audiogridder#1099] [Help] [answered]
sorry for my english (google translator dont help....)

In normal situation in Cubase, tab look like this 
![image](https://user-images.githubusercontent.com/51825613/203367169-1aa89f9a-73bc-443e-9d17-692a32ce1d63.png)

All working fine with Cubase in houses between 1 km:)

But in situation with FL Studio i have a problem with signal distortion and etc. Tab looking like this
![image](https://user-images.githubusercontent.com/51825613/203367899-8ec424f3-da9e-442e-8084-87577636af7f.png)

All working bad with glitches distortion.

I go deeper and check settings of plugin in FL Studio wrapper and found so many connections
![image](https://user-images.githubusercontent.com/51825613/203368473-b80e698c-1ccf-4232-8edb-ad929c166f31.png)

Its not good

For example usual plugin look like this
![image](https://user-images.githubusercontent.com/51825613/203368769-d582788d-fbff-4d06-bd8e-53176d032b45.png)

The main problem is due to the fact that the Audio Reader plugin launched in fl studio transmits a lot of channels at once and there are a lot of distortions in the sound. And if you take Cubase, there is no such problem because of the small number of channels and the sound turns out smooth

**Comment by apohl79** [comment]
If you disable the channels you don't need, does that help?

**Reply by videohosting** [reply]
If i disable any 1 output channel in AGridder plugin, the sound disappears

If i disable in FL Studio plugin settings, sound still working, but so many pauses and crackles (Without activated plugins,only empty audiogridder)
But other 3rd party plugins in FL Studio don`t create so many channels and working without glitches crackles etc.

In other daw all fine

**Reply by videohosting** [reply]
Yes if i try disable channel sound disappears...

**Reply by videohosting** [reply]
![image](https://user-images.githubusercontent.com/51825613/203563660-7396b17c-9c60-4f3d-8f78-6c509d6aa23e.png)
Instruments working good after setup good buffer size and enable only main channels


But if i trying disable any output channel in effect mode, the sound does not get into the plugin itself and passes without processing, and sometimes disappears altogether

**Reply by videohosting** [reply]
As a result: in FL Studio, it works well in the musical instrument mode, but it no longer works in the effect mode...

**Comment by videohosting** [comment]
As a result: in FL Studio, it works well in the musical instrument mode, but it no longer works in the effect mode...

**Comment by videohosting** [comment, marked as answer]
I found answer, need use VST3

I use VST and have this problem,but i try disable merge VST and VST3 in FL Studio plugin merge,run only VST3 and its working fine now!

**Reply by apohl79** [reply]
thx for this info!
